### PR TITLE
Client7 cont

### PR DESCRIPTION
**Pull request by 31453**

### CODE REVIEW

File: viewer/viewer.js [hunk 1]
```diff
@@ -2418,20 +2417,25 @@ internals.processCronQueries = () => {
...
   }
 
   Db.checkVersion(MIN_DB_VERSION, Config.get('passwordSecret') !== undefined);
-  Db.healthCache(function (err, health) {
+
+  try {
+    const health = await Db.healthCache();
     internals.clusterName = health.cluster_name;
-  });
+  } catch {}
```
[the remark this answers is not in the document]
Comment (awick) [reply viewer/viewer.js:2430]: Sure log, might need to exit, not sure if it can ever actually happen

[the remark this answers is not in the document]
Comment (31453) [reply viewer/viewer.js:2430]: I think it is VERY unlikely to happen and will only happen if something is very wrong with elasticsearch. 

Previously we were doing nothing if it failed. I'm just going to log it.

File: viewer/db.js [hunk 2]
```diff
@@ -1004,100 +1015,78 @@ exports.molochNodeStatsCache = function (nodeName, cb) {
...
-      return cb(err, null);
-    }
-
-    internals.elasticSearchClient.indices.getTemplate({ name: fixIndex('sessions2_template'), filter_path: '**._meta' }, (err, doc) => {
-      if (err) {
-        return cb(null, health);
-      }
+  try {
+    const health = await exports.health();
+    try {
```
[the remark this answers is not in the document]
Comment (31453) [reply viewer/db.js:1025]: No, I do need the nested try here to return health if the second try fails

**Comment by Arkadhan** [comment]
👍

**Pull request merged**